### PR TITLE
Get icon androguard python3

### PR DESCRIPTION
**Pull request by superpoussin22**
use Androguard to get icon , sounds to solve icon issue in particular on mac

### CODE REVIEW

File: StaticAnalyzer/views/android/icon_analysis.py [hunk 1]
```diff
@@ -74,31 +75,21 @@ def guess_icon_path(res_dir):
...
-            aapt_output = subprocess.check_output(args)
-        regex = re.compile(r"application:[^\n]+icon='(.*)'.*")
-        found_regex = regex.findall(aapt_output.decode('utf-8'))
-        if len(found_regex) > 0:
-            if found_regex[0]:
-                return {
-                    'path': os.path.join(os.path.dirname(apk_path), found_regex[0]),
-                    'hidden': False
-                }
+         a = apk.APK(apk_path)
```
Comment (ajinabraham) [inline StaticAnalyzer/views/android/icon_analysis.py]: Unnecessary space here. Please Correct the Indentation.

**Comment by superpoussin22** [comment]
done

**Comment by ajinabraham** [comment]
LGTM. Merging and Updating few things.

**Comment by ajinabraham** [comment]
Fixed https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/536

**Pull request merged**